### ISSUE TITLE
Powerlevel9k not starting properly when loading from /etc/zshrc

### ISSUE THREAD

**Issue by cawilliamson** [bhilburn/powerlevel9k#911]
The problem I'm seeing is when I have the following in my zshrc file the theme does not load properly:

```
zplug "bhilburn/powerlevel9k", as:theme, defer:2
zplug load
```

I see the following as my prompt:

`$(build_left_prompt)                                     $(build_right_prompt)$reset_color`

Strangely, however, if I then run zplug load again the theme loads properly and all is well.

The same applies with the following antibody code:

```
source <(antibody init)
antibody bundle bhilburn/powerlevel9k
```

Any ideas? Not sure what I'm missing here - I have also tried the documented install method for zplug and get the same results. In both cases - if I run the module management tool manually after this the theme loads properly with either tool.

**Comment by dritter** [comment]
Hi @cawilliamson !
Thanks for the report. We recently released `0.6.5`. Is that something you encounter since then, or did you have the issue with the previous release?

Btw. Our [installation guide]() states for zplug it should be `zplug "bhilburn/powerlevel9k", use:powerlevel9k.zsh-theme`. Not sure what is the difference between `use` and `theme`.. Could you try that and see if that helps?

**Comment by bhilburn** [comment]
Closing this one due to inactivity. If you are still having issues, please re-open and let us know!
